tools/congestion-model/src/model/queue_bundle.rs

```diff
@@ -23,17 +23,17 @@ impl QueueBundle {
         };
 
         for &shard in shards {
-            let mailbox = this.new_queue(shard);
+            let mailbox = this.new_queue(shard, "mailbox");
```

Comment:
What the heck is mailbox? :)

Reply:
Borrowed vocabulary from the actor model

https://www.brianstorti.com/the-actor-model/

The model basically treats each shard as an actor and the implicit incoming receipts queue is the actor's mailbox. And that's no coincidence. I always like to map problems to solutions that already exist.

Not only is the model implemented this way, I think even the real implementation can be thought of this way. Taking inspiration from Erlang, Akka, Actix, and other established actor frameworks could be useful. But it's also quite limited since they generally react to full queues by dropping messages / returning failures (which I still think we can avoid) and they don't operate on a global round-based clock that the blockchain has thanks to block heights. I think those are the main differences in constraints.

Anyway, sorry for the off-topic rambling... "mailbox" as the queue name makes sense in my mind but open to other names :)